version/version.go

```diff
@@ -13,7 +15,7 @@ var reVersion = regexp.MustCompile(`^(v\d+\.\d+.\d+)(?:-)?(.+)?$`)
 func String() string {
 	matches := reVersion.FindStringSubmatch(GitVersion)
 	if matches == nil || len(matches) < 3 {
-		return "v0.0.0+" + GitVersion
+		return ReleaseVersion + "+" + GitVersion
```

Comment:
This is incorrect. This line is only encountered when there is no git tag on the repo. We should tag our versions properly.

Reply:
Oops. Thanks for raising. Will revert and manage via git tags.